.github/workflows/ci.yml

```diff
@@ -452,13 +452,14 @@ jobs:
       - name: Install X11 libraries (Linux)
         if: ${{ startsWith(matrix.os, 'ubuntu') }}
         run: |
-          sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 x11-utils xvfb
+          sudo DEBIAN_FRONTEND=noninteractive apt install libegl1 libopengl0
+          sudo DEBIAN_FRONTEND=noninteractive apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 x11-utils xvfb
 
       - name: Try running the installation (Linux)
         if: ${{ startsWith(matrix.os, 'ubuntu' )}}
         # If sasview has crashed on its own, then this should return an error
         run: |
-          xvfb-run -a --server-args="-screen 0 1024x768x24" $RUN_SASVIEW &
+          xvfb-run -e /dev/stdout -a --server-args="-screen 0 1024x768x24" $RUN_SASVIEW &
```

Comment:
My conclusion was that a lot more was needed here - I've been playing and have put that in PR #3070

Reply:
I think your work might supersede this PR, but I need to take a better look. If so, I'll close this.